presentation/src/main/java/me/androidbox/presentation/login/viewmodel/LoginViewModel.kt

```diff
@@ -18,7 +18,7 @@ class LoginViewModel @Inject constructor(
     private val loginUseCase: LoginUseCase
 ) : ViewModel() {
 
-    private val loginScreenMutableState = MutableStateFlow(AuthenticationScreenState<Login>("", "", false, "",null))
+    private val loginScreenMutableState = MutableStateFlow(AuthenticationScreenState<LoginUser>("", "", false, "",null))
```

Comment:
I personally prefer to set the default values of the state in the state itself and not here in the ViewModel. That way you can also easily reset the state by doing `_state.update { AuthenticationScreenState() }` without needing to reset each field manually

Reply:
@philipplackner Updated to have default values